tests/system/test_system.py

```diff
@@ -1751,6 +1751,49 @@ def test_compose_with_generation_match(self):
         composed = original.download_as_bytes()
         self.assertEqual(composed, BEFORE + TO_APPEND)
```

Comment:
@tseaver Re: Do we need a system test for the deprecated case?
`test_compose_with_generation_match_list` tests both deprecated cases that would fail or succeed. Here we test operation to succeed with a deprecated case in which list of long is passed in and source objects generations match. Are there any other cases missing?

Reply:
I was actually questioning the need for systests for the deprecated case(s), rather than arguing for more. :)